src/index.ts

```diff
@@ -37,7 +37,6 @@ const main = async () => {
   bot.use(
     createContextMiddleware({
       config,
-      logger: logger.child({ source: 'database' }),
```

Comment:
this is not related to this PR, but made sense to be removed from this signature